keyboards/sparrow62/keymaps/74th/keymap.c

```diff
@@ -0,0 +1,88 @@
+    // |-------+-------+-------+-------+-------+-------+-------| |-------+-------+-------+-------+-------+-------+-------|
+    // |Sft(ESC|   Z   |   X   |   C   |   V   |   B   |   [   | |   ]   |   N   |   M   |  , <  |  . >  |  / ?  |  \ |  |
+    SFT_T(KC_ESC),KC_Z,  KC_X,   KC_C,   KC_V,   KC_B,  KC_LBRC,  KC_RBRC, KC_N,   KC_M,  KC_COMM,KC_DOT, KC_SLSH,KC_BSLS,
+    // |-------+-------+-------+-------+-------+-------+-------| |-------+-------+-------+-------+-------+-------+-------|
+    //                       |  ESC  |Alt(TAB|Sf(Eisu|Ctl(SPC|     |Ctl(ENT|  FN   |GU(Kana|  BS   |
+    ESC_MHEN, ALT_T(KC_TAB), SFT_T(KC_MHEN), CTL_T(KC_SPC), CTL_T(KC_ENT), MO(_FN), GUI_T(KC_HENK), KC_BSPC
+    //                       \-------+-------+-------+-------/     \-------+-------+-------+-------/
+    ),
+
+    [_FN] = LAYOUT(
+    // /-------+-------+-------+-------+-------+-------\                 /-------+-------+-------+-------+-------+-------.
+    // | Lock  |CMD+F1 |Sft+F2 |CMD+F3 |CMD+F4 |CMD+F5 |                 |CMD+F6 |CMD+F7 |CMD+F8 |CMD+F9 |       |       |
+    LOCK,LCMD(KC_F1),SCMD(KC_F2),LCMD(KC_F3),LCMD(KC_F4),LCMD(KC_F5),LCMD(KC_F6),LCMD(KC_F7),LCMD(KC_F8),LCMD(KC_F9),_______,_______,
+    // |-------+-------+-------+-------+-------+-------+-------\ /-------+-------+-------+-------+-------+-------+-------|
+    // | ESC   |  F1   |  F2   |  F3   |  F4   |  F5   |  F6   | |  F6   |  F7   |  F8   |  F9   |  F10  |  F11  |  F12  |
+        KC_ESC,  KC_F1,  KC_F2,  KC_F3,  KC_F4,  KC_F5,  KC_F6,    KC_F6,  KC_F7,  KC_F8,  KC_F9, KC_F10, KC_F11, KC_F12,
+    // |-------+-------+-------+-------+-------+-------+-------| |-------+-------+-------+-------+-------+-------+-------|
+    // |   ~   |   !   |   @   |   #   |   $   |   %   |   ^   | |   ^   |   &   |   *   |       | PrSC  | PrSC2 | PrSC3 |
+        KC_GRV, KC_EXLM, KC_AT, KC_HASH,KC_DLR, KC_PERC,KC_CIRC,  KC_CIRC,KC_AMPR,KC_ASTR,_______,KC_PSCREEN,LALT(KC_PSCREEN),LSFT(KC_PSCREEN),
+    // |-------+-------+-------+-------+-------+-------+-------| |-------+-------+-------+-------+-------+-------+-------|
+    // |       |       |       |       |       |       |       | |       |   ←   |   ↓   |   ↑   |   →   | HOME  |  END  |
+        _______,_______,_______,_______,_______,_______,_______,  _______,KC_LEFT,KC_DOWN, KC_UP,KC_RIGHT,KC_HOME,KC_END,
+    // |-------+-------+-------+-------+-------+-------+-------| |-------+-------+-------+-------+-------+-------+-------|
+    //                       |  ESC  |Alt(TAB|Sf(Eisu|Ctl(SPC|     |       |  FN   | RAlt  |  DEL  |
+                              _______,_______,_______,_______,      _______,_______,_______,KC_DEL
+    //                       \-------+-------+-------+-------/     \-------+-------+-------+-------/
+    )
+};
+
+bool process_record_user(uint16_t keycode, keyrecord_t *record) {
+    if (record->event.pressed) {
+        switch (keycode) {
+            case LOCK:
+                tap_code16(C(G(KC_Q)));
```

Comment:
you can actually use the `C(G(KC_Q))` in the keymap directly, you don't need a macro for this.

Reply:
That's right. I fixed it that way.